rust/core-lib/src/rpc.rs

```diff
@@ -322,6 +322,17 @@ pub struct MouseAction {
     pub click_count: Option<u64>,
 }
 
+/// Represents how the current selection is modified (used by find
+/// operations).
+#[derive(Serialize, Deserialize, PartialEq, Eq, Debug)]
+#[serde(rename_all = "snake_case")]
+pub enum SelectionModifier {
+    None,
```

Comment:
In your follow on PR about the Option/None stuff, it'd be nice if we could add an explanation for each of these cases.